src/main/java/com/ontotext/trree/plugin/mongodb/MongoDBPlugin.java

```diff
@@ -42,6 +42,7 @@
 	public static final IRI HINT = SimpleValueFactory.getInstance().createIRI(NAMESPACE + "hint");
 	public static final IRI ENTITY = SimpleValueFactory.getInstance().createIRI(NAMESPACE + "entity");
 	public static final IRI GRAPH = SimpleValueFactory.getInstance().createIRI(NAMESPACE + "graph");
+	public static final IRI COLLATION = SimpleValueFactory.getInstance().createIRI(NAMESPACE + "collate");
```

Comment:
It might be better to call this "collation" instead of "collate" to match the form used in the MongoDB API.